zebra-state/src/service/finalized_state/disk_format/upgrade/add_subtrees.rs

```diff
@@ -0,0 +1,181 @@
+//! Adds Sapling and Orchard note commitment subtrees to database
```

Comment:
Nit: The previous PR adds the subtrees, this PR fully populates them

```suggestion
//! Fully populate the Sapling and Orchard note commitment subtrees for existing blocks in the database.
```